Fixes #4654 - Max Zoom and Min Zoom shortcuts

leonardochappuis opened the pull request:
Fixes #6291

**Changes**
Adds two new functions (zoom to max and zoom to min) and their respective keyboard shortcuts -- called with Shfit + + and Shift + -, respectively.
Also adds tooltips for them in minimap menu.
Do let me know if you don't these shortcuts so I can alter them.

leonardochappuis committed:
Add zoom to max and zoom to min functions/shortcuts

Commit:
6 files were automatically formatted.

tothxa reviewed (commented):
I think this is what will make it work as intended. (not tested, only based on increase/decrease. I didn't remember View.zoom was backwards...)

leonardochappuis committed:
Change order of zoom functions, remove extra tips

leonardochappuis reviewed (commented):
Changed it in new commit as requested. Changes tested locally.

Commit:
4 files were automatically formatted.

leonardochappuis committed:
Add back tips to zoom max and zoom in

leonardochappuis reviewed (commented):
Added back tips as requestes

leonardochappuis committed:
Change user facing strings (zoom max/min -> maxium minimum)

knarrff committed:
Let max/min zoom actions toggle max/min zoom

This patch lets the max/min zoom actions (keys, menu) go to previously
saved zoom value if already at max/min zoom level. This makes it
possible to quickly zoom out/in to min/max zoom level and back to
whatever zoom level was used before.

Commit:
2 files were automatically formatted.

leonardochappuis committed:
Merge branch 'master' into zoom-max-min-add

Noordfrees committed:
Update for master merge

Noordfrees committed:
Merge commit 'ddafe53b53533442e961a32a299fef1e3d6f188c' into HEAD

Noordfrees committed:
Merge branch 'master' of codeberg.org:wl/widelands into HEAD
